distrobuilder/main.go

```diff
@@ -190,6 +193,54 @@ func run(c *cli.Context) error {
 	// Unmount everything and exit the chroot
 	exitChroot()
 
+	if c.GlobalBool("lxc") {
+		img := image.NewLXCImage(c.GlobalString("cache-dir"), def.Image, def.Targets.LXC)
+
+		for _, file := range def.Files {
+			generator := generators.Get(file.Generator)
+			if generator == nil {
```

Comment:
Is that the case where the generator doesn't exist? If so, shouldn't that error out instead?